src/sentry/snuba/dataset.py

```diff
@@ -73,6 +73,10 @@ class EntityKey(Enum):
     GenericMetricsSets = "generic_metrics_sets"
     GenericMetricsCounters = "generic_metrics_counters"
     GenericMetricsGauges = "generic_metrics_gauges"
+    GenericMetricsDistributionsMeta = "generic_metrics_distributions_meta"
```

Comment:
Why did you add these? I don't think the meta tables should be queryable outside of the metrics layer.

Reply:
The thing is that this entity key can be queried nonetheless so I would prefer to centralized its definition. The same was done for generic org counters, they are not needed outside of dynamic sampling but they are defined here for clarity.

Reply:
The only place these entities are queried is the metrics layer. Adding them here implies that querying them outside the metrics layer is an allowed practice, and I don't think it should be.